Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM openjdk:26-ea-trixie
+EXPOSE 8080
+ADD backend/target/app.jar app.jar
+ENTRYPOINT ["java", "-jar", "app.jar"]
```

Comment:
i mean that kinda works.

BUT, i would use a multistage docker file.

1st Stage:
FROM node:latest as frontend

// build /dist

2nd Stage
FROM openjdk:26-whatever as backend
COPY --from=frontend /dist

/* 
build your jar here
*/

ENTRYPOINT ["java", "-jar", "app.jar"]